### ISSUE TITLE
ImageStore deprecated

### ISSUE THREAD

**Issue by tronze** [GeekyAnts/vue-native-core#161]
```
Warning: ImageStore is deprecated and will be removed in a future release. To get a base64-encoded string from a local image use either of the following third-party libraries:* expo-file-system: `readAsStringAsync(filepath, 'base64')`* react-native-fs: `readFile(filepath, 'base64')`
- node_modules/expo/build/environment/muteWarnings.fx.js:26:24 in error
- node_modules/react-native/node_modules/fbjs/lib/warning.js:30:18 in printWarning
- node_modules/react-native/node_modules/fbjs/lib/warning.js:51:23 in <unknown>
- node_modules/react-native/Libraries/Utilities/warnOnce.js:29:10 in warnOnce
- node_modules/react-native/Libraries/react-native/react-native-implementation.js:54:6 in ImageStore
- node_modules/vue-native-helper/build.js:3942:39 in <unknown>
- node_modules/vue-native-helper/build.js:3939:33 in buildNativeComponent
* App.vue:104:74 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:331:6 in loadModuleImplementation
- node_modules/expo/AppEntry.js:4:0 in <unknown>
- node_modules/metro/src/lib/polyfills/require.js:331:6 in loadModuleImplementation
- node_modules/metro/src/lib/polyfills/require.js:197:45 in guardedLoadModule
* null:null in global code
```

**Comment by drsaint23** [comment]
Getting exact error, started to show up when I've upgraded to Expo 33. Since I'm not using ImageStore anywhere in the code,  I think it might be related to AsyncStorage

**Comment by tronze** [comment]
> Getting exact error, started to show up when I've upgraded to Expo 33. Since I'm not using ImageStore anywhere in the code, I think it might be related to AsyncStorage

**Comment by tronze** [comment]
Yes, I also think it is related to AsyncStorage. Since I'm not sure where does ImageStore is used in the system, I'm looking at the code and trying to update before it is actually deprecated

**Comment by qodehub-nanadjei** [comment]
Having the same error.

**Comment by tim92109** [comment]
Same here... I'll add that it throws with every code change (build).  Very distracting when trying to debug.

**Comment by tronze** [comment]
> Same here... I'll add that it throws with every code change (build). Very distracting when trying to debug.

Yes.. I think that Vue Native is using deprecated libraries which means even I don't use ImageStore in my app, it will throw this warning message all the time because the deprecated libraries will be included in app building process. (Found it happening when Vue Native is transforming into React Native in app building process.)

I think the solution will be updating the deprecated library, but I'm still struggling to find out what deprecated libraries Vue Native using.

**Comment by icarter09** [comment]
I know it's not a good long term fix, but I commented out the warning message. It can be found in `node_modules/react-native/Libraries/react-native/react-native-implementation.js` lines 52-60. I commented out the `warnOnce()` which is where the error message is being returned. 
Like I said, not a good long term fix, but helped me to continue development and debugging. I'm currently looking into a better fix.
